var/spack/repos/builtin/packages/acts/package.py

```diff
@@ -124,6 +125,7 @@ class Acts(CMakePackage, CudaPackage):
     # FIXME: Cannot build ONNX plugin as Spack doesn't have an ONNX runtime
     # FIXME: Cannot build SyCL plugin yet as Spack doesn't have SyCL support
```

Comment:
Have you checked out https://github.com/spack/spack/blob/develop/var/spack/repos/builtin/packages/hipsycl/package.py, which provides the `sycl` interface?

The following packages have `sycl` variants:
- `arborx` (via `kokkos`)
- `gromacs`
- `kokkos`

`intel-llvm` has a `sycl` version while the following have multi-valued variants that include `sycl` for their compute backends:
- `hipace`
- `warpx`

Reply:
I somehow missed that at the time I wrote that comment, strange... Anyway, I'm trying that out.